python/pypto/language/op/tile_ops.py

```
@@ -987,19 +987,32 @@ def min(tile: Tile | Scalar | int, axis: int | Scalar = 0, keepdim: bool = False
     return Tile(expr=call_expr)
 
 
-def slice(tile: Tile, shape: Sequence[IntLike], offset: Sequence[IntLike]) -> Tile:
-    """Create a slice of a tile with new shape and offset.
+def slice(
```

Ruff (0.15.6)

[error] Variable `slice` is shadowing a Python builtin

(A001)